### PR TITLE
Login page converted to mudblazor

### PR DESCRIPTION
**Pull request by Frederikmoeller**

### CODE REVIEW

File: frontend/Pages/Login.razor [hunk 1]
```diff
@@ -1,19 +1,35 @@
+@using MudBlazor
+@using System.ComponentModel.DataAnnotations
+@inject ISnackbar Snackbar
```
Comment (Mightyhaha) [inline frontend/Pages/Login.razor:3]: We inject snackbar, but we are not using it?

Comment (CasperStaahl) [reply frontend/Pages/Login.razor:3]: ye who cares

File: frontend/Pages/Login.razor [hunk 2]
```diff
@@ -1,19 +1,35 @@
...
+        </MudItem>
+    </MudGrid>
+</MudContainer>
+
+@code
+{
+    bool success;
+    string[] errors = { };
+    MudTextField<string> pwFieldLogin;
+    MudForm loginForm;
```
Comment (CasperStaahl) [inline frontend/Pages/Login.razor:34]: please use explicit access modifiers and prefix private fields with _

**Comment by CasperStaahl** [comment]
I would like the login form to be moved at bit down, so it's not i contact with the topbar.

**Review by CasperStaahl — CHANGES_REQUESTED** [review]

**Review by CasperStaahl — APPROVED** [review]

**Pull request merged**